nistats/design_matrix.py

```diff
@@ -450,6 +450,54 @@ def plot_design_matrix(design_matrix, rescale=True, ax=None):
     return ax
 
 
+def create_simple_second_level_design(maps_table, confounds=None,
```

Comment:
In reality this should be the only function and eliminate the previous one, I wanted to ask you if you think the previous design matrix generation would be useful in the future or if I should just wipe it out.